lib/relationships/relationships.ts

```diff
@@ -0,0 +1,279 @@
+import * as _ from 'lodash';
```

Comment:
What's the reason for this file being at `lib/relationships/relationships.ts` instead of `lib/relationships.ts`?

Reply:
Ditto